cpu/cortexm_common/include/cpu.h

```diff
@@ -40,6 +40,14 @@ extern "C" {
 #endif
 
 /**
+ * @brief Interrupt stack canary value
+ *
+ * @note 0xe7fe is the ARM Thumb machine code equivalent of asm("bl #-2\n") or
+ * 'while (1);', i.e. an infinite loop.
+ */
+#define STACK_CANARY_WORD 0xE7FEE7FEu
```

Comment:
I didnt redefine it, I moved it to 'cup.h' since it's included in both vectors_cortex.c and thread_arch.c, wouldn't that be ok?